src/primitive_events.cc

```diff
@@ -53,4 +53,15 @@ PRIMITIVE(register_object_notifier) {
   return process->program()->null_object();
 }
 
+PRIMITIVE(unregister_object_notifier) {
+  ARGS(ByteArray, group_proxy, ByteArray, resource_proxy);
```

Comment:
You can just declare these as ResourceGroup and Resource.

Reply:
Not if they are closed.